Symfony/CS/Fixer.php

```diff
@@ -242,7 +250,11 @@ public function fixFile(\SplFileInfo $file, array $fixers, $dryRun, $diff, FileC
                 FixerFileProcessedEvent::create()->setStatus(FixerFileProcessedEvent::STATUS_EXCEPTION)
             );
 
-            $this->errorsManager->report(ErrorsManager::ERROR_TYPE_EXCEPTION, $this->getFileRelativePathname($file), $e->__toString());
+            $this->errorsManager->report(new Error(
+                Error::TYPE_FIXING,
+                $this->getFileRelativePathname($file),
+                $e->__toString()
```

Comment:
@keradus 

I only changed the wrapping here, but happy to adjust. Also see https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1102/files#r28657009.